### PR TITLE
All pupil memory citations

### PR DESCRIPTION
**Pull request by KirstensGitHub**
### List of citation keys added (one per line)

AherBeat81

BeukEtal19

BoisEtal16

DetrMa16

EinhEtal08

Fabe17

FiedGloc12

GardEtal74

GardEtal75

GoddBadd75

GoldPape12

HeavHutt11

Jaco91

JohnEtal93

KafkEtal15

KahnBeat66

KahnEtal67

KrueTian04

LittEtal11

Luke2017

Mand80

Mill2016

MontEtal18

NabeEtal13

OterEtal11

PapeEtal12

Rapc19

Satt46

Tulv02

Tulv85

VilbRugg08

VoEtal07

YoneAndr02

ZekvEtal18

### CODE REVIEW

File: memlab.bib [hunk 1]
```diff
@@ -9852,6 +9853,15 @@ @incollection{MannEtal15
 	Title = {The role of context in episodic memory},
 	Year = {2015}}
 
+@incollection{AherBeat81,
+    Title = {{Physiological Evidence that Demand for Processing Capacity Varies with Intelligence}},
+    Year = {1981},
+    Booktitle = {Intelligence and Learning},
+    Author = {Ahern, S and Beatty, J},
+    Pages = {121-128},
```
Comment (jeremymanning) [inline memlab.bib:9861]: Can you use double dashes for page ranges?  E.g. this should be "121--128"

File: memlab.bib [hunk 2]
```diff
@@ -9930,6 +9940,42 @@ @article{GemaGema84
...
 	Bdsk-Url-1 = {http://dx.doi.org/10.2307/2290921}}
 
+@article{BeukEtal19,
+    Title = {{The Pupillary Response to the Unknown: Novelty Versus Familiarity}},
+    Year = {2019},
+    Journal = {i-Perception},
+    Author = {Beukema, S. and Jennings, B. J. and Olson, J. A. and Kingdom, F. A. A.},
+    Number = {5},
+    Volume = {10},
+		Pages={2041669519874817},
```
Comment (jeremymanning) [inline memlab.bib:9950]: This seems unlikely to be the correct page...can you double check it?

File: memlab.bib [hunk 3]
```diff
@@ -9930,6 +9940,42 @@ @article{GemaGema84
...
+    Doi = {10.1016/j.neubiorev.2016.05.034},
+}
+
+@article{DetrMa16,
+    Title = {{Analyzing repeated measurements using mixed models}},
+    Year = {2016},
+    Journal = {Journal of the American Medical Association},
+    Author = {Detry, M. A. and Ma, Y.},
+    Number = {4},
+    Pages = {407-408},
```
Comment (jeremymanning) [inline memlab.bib:9972]: double dash

File: memlab.bib [hunk 4]
```diff
@@ -9966,6 +10012,17 @@ @article{BindEtal12
 	Volume = {63},
 	Year = {2012}}
 
+@article{EinhEtal08,
+  Title = {{Pupil dilation reflects perceptual selection and predicts subsequent stability in perceptual rivalry}},
+  Year = {2008},
+  Journal = {Proceedings of the National Academy of Sciences},
+  Author = {Einh{\"{a}}user, W. and Stout, J. and Koch, C. and Carter, O.},
+  Number = {5},
+  Pages = {1704-1709},
```
Comment (jeremymanning) [inline memlab.bib:10021]: double dash

File: memlab.bib [hunk 5]
```diff
@@ -9978,13 +10035,26 @@ @article{KensScha06
 	Volume = {6},
 	Year = {2006}}
 
+@article{Fabe17,
+  Title = {{Neuromodulation of pupil diameter and temporal perception}},
+  Year = {2017},
+  Journal = {Journal of Neuroscience},
+  Author = {Faber, N. J.},
+  Number = {11},
+  Pages = {2806-2808},
```
Comment (jeremymanning) [inline memlab.bib:10044]: double dash

File: memlab.bib [hunk 6]
```diff
@@ -9997,6 +10067,18 @@ @article{ErkEtal05
 	Volume = {26},
 	Year = {2005}}
 
+@article{FiedGloc12,
+  Title = {{The Dynamics of Decision Making in Risky Choice: An Eye-Tracking Analysis}},
+  Year = {2012},
+  Journal = {Frontiers in Psychology},
+  Author = {Fiedler, S. and Gl{\"{o}}ckner, A.},
+  Number = {OCT},
+  Pages = {335},
```
Comment (jeremymanning) [inline memlab.bib:10076]: just one page?

File: memlab.bib [hunk 7]
```diff
@@ -9997,6 +10067,18 @@ @article{ErkEtal05
 	Volume = {26},
 	Year = {2005}}
 
+@article{FiedGloc12,
+  Title = {{The Dynamics of Decision Making in Risky Choice: An Eye-Tracking Analysis}},
```
Comment (jeremymanning) [inline memlab.bib:10071]: remove outer set of curly braces in this title

File: memlab.bib [hunk 8]
```diff
@@ -10573,6 +10655,99 @@ @article{SimmEtal07
 	Volume = {45},
 	Year = {2007}}
 
+@article{GardEtal74,
+  Title = {{Inhibition of pupillary orienting reflex by novelty in conjunction with recognition memory}},
```
Comment (jeremymanning) [inline memlab.bib:10659]: remove outer set of curly braces

File: memlab.bib [hunk 9]
```diff
@@ -10573,6 +10655,99 @@ @article{SimmEtal07
...
+  Year = {1974},
+  Journal = {Bulletin of the Psychonomic Society},
+  Author = {Gardner, R. M. and Mo, S. S. and Borrego, R.},
+  Number = {3},
+  month = {11},
+  Pages = {237-238},
```
Comment (jeremymanning) [inline memlab.bib:10665]: double dash

File: memlab.bib [hunk 10]
```diff
@@ -10573,6 +10655,99 @@ @article{SimmEtal07
...
+  Volume = {3},
+  Publisher = {Springer},
+  Doi = {10.3758/BF03333458}
+}
+
+@article{GardEtal75,
+  Title = {{Pupillary changes during encoding, storage, and retrieval of information}},
```
Comment (jeremymanning) [inline memlab.bib:10672]: remove outer set of curly braces (in this and following titles)

File: memlab.bib [hunk 11]
```diff
@@ -10573,6 +10655,99 @@ @article{SimmEtal07
...
+  Year = {1975},
+  Journal = {Perceptual and Motor Skills},
+  Author = {Gardner, R. M. and Beltramo, J. S. and Krinsky, R.},
+  Number = {3},
+  Pages = {951-955},
```
Comment (jeremymanning) [inline memlab.bib:10677]: add double dash (in this and following page ranges)

File: memlab.bib [hunk 12]
```diff
@@ -19365,6 +19540,223 @@ @article{TroyEtal97
...
+  Journal = {Science},
+  Author = {Kahneman, D. and Beatty, J. and Pollack, I.},
+  Number = {3785},
+  Pages = {218-219},
+  Volume = {157},
+  Publisher = {American Association for the Advancement of Science},
+}
+
+@article{KrueTian04,
+  Title = {{A comparison of the general linear mixed model and repeated measures ANOVA using a dataset with multiple missing data points}},
```
Comment (jeremymanning) [inline memlab.bib:19566]: for this title it should be `{A comparison of the general linear mixed model and repeated measures {ANOVA} using a dataset with multiple missing data points}`

File: memlab.bib [hunk 13]
```diff
@@ -19365,6 +19540,223 @@ @article{TroyEtal97
...
+  Volume = {46},
+  Publisher = {Pergamon},
+  Doi = {10.1016/j.neuropsychologia.2008.01.004},
+}
+
+@article{VoEtal07,
+  Title = {{The coupling of emotion and cognition in the eye: Introducing the pupil old/new effect}},
+  Year = {2007},
+  Journal = {Psychophysiology},
+  Author = {Võ, M. L.-H. and Jacobs, A. M. and Kuchinke, L. and Hofmann, M. and Conrad, M. and Schacht, A. and Hutzler, F.},
```
Comment (jeremymanning) [inline memlab.bib:19728]: Võ should be: "V\"{o}"

**Review by jeremymanning — CHANGES_REQUESTED** [review]
a few formatting requests...

File: memlab.bib (hunk 2 above)
Comment (KirstensGitHub) [reply memlab.bib:9950]: Weird, it's hard to tell! Here is the google scholar citation:

image

Here is the paper:
https://journals.sagepub.com/doi/full/10.1177/2041669519874817

I don't see page number anywhere...!?

File: memlab.bib (hunk 1 above)
Comment (KirstensGitHub) [reply memlab.bib:9861]: got it!

File: memlab.bib (hunk 2 above)
Comment (KirstensGitHub) [reply memlab.bib:9950]: image

**Comment by jeremymanning** [comment]
hey @KirstensGitHub can you correct the merge conflicts so that i can merge your pull request?

**Comment by jeremymanning** [comment]
(the changes look good aside from the conflicts)

**Comment by KirstensGitHub** [comment]
i tried to fix them using atom and it's not showing other merge conflicts that I can see but the pull request still says there are conflicts? I can't use the "Resolve conflicts" button, which I guess might show what the issue is?

**Comment by jeremymanning** [comment]
@KirstensGitHub did you pull the latest changes when you did the merge?  If so, I think you can just commit and push to your branch and the pull request should update...

**Pull request merged**